node/actors/network/src/gossip/testonly.rs

```diff
@@ -1,7 +1,7 @@
 #![allow(dead_code)]
 use super::*;
 use crate::{frame, mux, noise, preface, rpc, Config, GossipConfig};
-use anyhow::Context as _;
+use anyhow::Context;
```

Comment:
?

Reply:
Sorry, that wasn't intentional. It's back to its original state